### DISCUSSION TITLE
3rd party libraries have security issues

### DISCUSSION THREAD

**Discussion by krop** [opentoonz/opentoonz#6450] [Q&A]
### Description

Since opentoonz still ships very old 3rd party libraries rather than using external dependencies, it is also responsible for the security issues in these copies.

The list of CVE issues is particularly long for unavoidable dependencies:
- libtiff: https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=libtiff
- lzo: https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=lzo

and also:
- tinyexr: https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=tinyexr
- zlib: https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=zlib
- lz4: https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=lz4
- libpng: https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=libpng
- libjpeg-turbo: https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=libjpeg-turbo

### Steps to Reproduce

1. 
2. 

### Expected Behavior

security fixes should be applied to third party libraries or they should be dropped completely

### OpenToonz Version

1.7

### Operating System

Linux

### GPU

NVIDIA

### Graphics Tablet

None

**Comment by RodneyBaker** [comment]
Converting this to a Discussion as it's a generally known area of interest that is continuously considered.  We do want to hone in more on specifics so each can be addressed.
